src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp.js';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const timeEntry=props.timeStamp
+
+  const onLikeButtonClick = () => {
+    const updatedChat = {
+        id: props.id,
+        sender: props.sender,
+        body: props.body,
+        timeStamp: props.timeStamp,
+        liked: !props.liked
+    };
+
+    props.onUpdateChat(updatedChat);
+  };
+
+  const heartColor = props.liked ? '❤️' : '🤍';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={props.chatEntry}>
```

Comment:
Instead of getting the class 'remote' or 'local' from props, we can calculate it here. 
```
<div className={props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local'}>